_rules/SC3-1-1-html-has-lang.md

```diff
@@ -25,7 +25,7 @@ The root element of the [page](https://www.w3.org/TR/WCAG21/#dfn-web-page-s), if
 
 ### Expectation
 
-The test target has a [non-empty](#non-empty) `lang` or `xml:lang` attribute.
+The test target has a `lang` or `xml:lang` attribute that is not empty ("").
```

Comment:
@kasperisager and @WilcoFiers, can you both agree to this change? 

> the empty string ("") --> empty ("")

If yes, I will just go ahead and make this change to all rules that uses that wording.